bin/collator/src/parachain/service.rs

```diff
@@ -729,12 +728,7 @@ pub async fn start_node(
     para_id: ParaId,
     additional_config: AdditionalConfig,
 ) -> sc_service::error::Result<(TaskManager, Arc<FullClient>)> {
-    let default_backend = parachain_config.chain_spec.default_network_backend();
-    // If the network backend is unspecified, use the default for the given chain.
-    let network_backend = parachain_config
-        .network
-        .network_backend
-        .unwrap_or(default_backend);
+    let network_backend = parachain_config.network.network_backend;
```

Comment:
I didn't check the stable2412 patch releases diff, but is there any other noteworthy changes other than this?